osquery/tables/system/authorized_keys.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <string>
+#include <vector>
+
+#include <osquery/core.h>
+#include <osquery/tables.h>
+#include <osquery/filesystem.h>
+#include <osquery/sql.h>
+
+namespace osquery {
+namespace tables {
+
+const std::vector<std::string> kSSHAuthorizedkeys = {
+    ".ssh/authorized_keys",".ssh/authorized_keys2"
+};
+
+void genSSHkeysForUser(const std::string& username,
+                            const std::string& directory,
```

Comment:
nitpick, To get well-formatted code use `git add .` then `make format` then `git commit --amend` when you make changes and are ready to push back to the PR with `git push -f`